iree/compiler/Dialect/HAL/Transforms/PackAllocations.cpp

```diff
@@ -126,12 +126,13 @@ class PackAllocationsPass
 
     Value offset = baseOffset;
     for (auto &slice : slices) {
-      auto sliceSize =
-          IREE::Util::align(loc, slice.dynamicSize, rangeAlignment, builder);
```

Comment:
There are more places you can update.

Remove these then patch up any places that called them:
https://github.com/google/iree/blob/a23ca45a5becbe55826cbcf2cb21aaac705796b7/iree/compiler/Dialect/Util/IR/UtilTypes.h#L190-L191
https://github.com/google/iree/blob/a23ca45a5becbe55826cbcf2cb21aaac705796b7/iree/compiler/Dialect/Util/IR/UtilTypes.cpp#L460-L474